src/components/images/header-logo-lg.tsx

```diff
@@ -2,32 +2,29 @@
 
 import {HtmlHTMLAttributes} from "react"
 import usePageHasTopBanner from "@lib/hooks/usePageHasTopBanner"
+import {useWindowSize} from "usehooks-ts"
+import {twMerge} from "tailwind-merge"
 
 type Props = HtmlHTMLAttributes<SVGSVGElement> & {
   height?: number
   width?: number
 }
 
-const HeaderLogoLg = ({height = 89, width = 579, ...props}: Props) => {
+const HeaderLogoLg = (props: Props) => {
   const hasPageBanner = usePageHasTopBanner()
-
-  if (height != 89 && width == 579) {
-    width *= height / 89
-  }
-
-  if (height == 89 && width != 579) {
-    height *= width / 579
-  }
+  const {width: windowWidth} = useWindowSize()
+  const height = windowWidth >= 1500 ? 126 : windowWidth <= 1500 ? 104 : 72
+  const width = (height / 89) * 579
+  const size = `w-[${width}px] h-[${height}px]`
```

Comment:
Dynamic classes don't work in tailwind
https://tailwindcss.com/docs/content-configuration#dynamic-class-names
Is there any way you can just do more simple classes without using the `useWindowSize`? Just some old fashion `md:w-[]` type classes?

Reply:
Most definitely! I'll update this shortly 😃